test/vega-lite/schema.json

```diff
@@ -27149,6 +27149,8 @@
     },
     "SingleDefUnitChannel": {
       "enum": [
+        "text",
```

Comment:
Somehow this type started getting out of order and breaking CI:

https://github.com/vega/ts-json-schema-generator/actions/runs/9105537200/job/25031273060#step:7:25

I changed its order so tests can pass, I'm also almost sure `enum` is order insensitive.